src/webgpu/web_platform/reftests/canvas_use_as_image_source_opaque.https.html

```diff
@@ -0,0 +1,32 @@
+<html class="reftest-wait">
```

Comment:
nit: perhaps put all of these in their own folder, and also clarify the name since these are all copying from webgpu canvases specifically. (`web_platform/reftests/webgpu_canvas_as_image_source/opaque.https.html` for example)